crates/rustc_codegen_spirv/src/codegen_cx/mod.rs

```diff
@@ -212,9 +212,16 @@ impl<'tcx> CodegenCx<'tcx> {
 
         // Create these up front instead of on demand in SpirvValue::def because
         // SpirvValue::def can't use cx.emit()
-        let global_var =
-            self.emit_global()
-                .variable(ty, None, StorageClass::Function, Some(initializer));
+        let global_var = self.emit_global().undef(ty, None);
```

Comment:
Is the change to the actual value needed to make this work?

Reply:
Oh, right, I don't think so no, accidentally included this because I was like "wait why isn't this OpVariable getting removed? uuh, let's try OpUndef? that didn't work either, hmm, oh right it's not getting zombied if it's not used" and then I never reverted it. I'll do so now.